### DISCUSSION TITLE
Adobe Air Runtime Download Gone

### DISCUSSION THREAD

**Discussion by G-Powder** [airsdk/Adobe-Runtime-Support#613] [Q&A]
Hi all.

Now that Adobe has officially handed AIR over to Harmon, I was wondering if there was still a link available to download the runtime only.  It seems to have been removed from Adobe's website and archives location.

We have a desktop application that has been around for several years and was it built using an older version of the SDK.
On occasion we need to re-install the runtime only for our existing users.

Any help would be appreciated.

Thanks.

**Comment by ajwfrost** [comment]
Our recommendation for folk would be to use a bundled runtime in their apps, but we are looking to provide a shared runtime still, for end users to download. Currently just trying to sort out the legal pieces needed for this, we have installers pretty much ready although facing a few issues still on Big Sur..

We should have things resolved and uploaded within the next two weeks I believe. If you still have that older SDK, you can rebuild your app with that AIR runtime bundled into it perhaps for any more urgent needs?

thanks

**Reply by jerome1981** [reply]
Hi there, any chance we could get an update on this?  Do you know when we should expect this standalone installer?  thanks.

**Reply by QAthulhu** [reply]
2 weeks have passed. Any update?

**Comment by QAthulhu** [comment]
why not make the old runtime installers available in the meantime? I can't wait 2 weeks

**Comment by spamshaker** [comment]
@ajwfrost 
> Our recommendation for folk would be to use a bundled runtime in their apps

Won't it require to sing the code with Apple developer ID by developers who recently delivered and signed code by adt
https://help.adobe.com/en_US/air/build/WS5b3ccc516d4fbf351e63e3d118666ade46-7f74.html
Will it be working on Mac OS when the code will be signed by adt with existing global code sign certificate without Apple ID?

**Comment by mrchnk** [comment]
I found AIR 32 beta available at [adobe labs](https://labs.adobe.com/downloads/air.html)

**Comment by ajwfrost** [comment]
FYI this is up now, apologies for the delay. Quite a few updates needed to work around the signing/notarization requirements on MacOS now..
https://airsdk.harman.com/runtime
One key outstanding issue is that we don't have validation of signed .air packages so everyone's app is currently treated the same as if it was signed with a self-signed certificate. We're looking into this ..


@spamshaker sorry I've only just seen your message there. Yes, if you are creating a bundled runtime app then you need to sign this with the appropriate Apple certificate, after creating the bundle. Essentially it becomes a two step process; on Windows, the equivalent mechanism allows you to pass in the additional signing details, but on MacOS that isn't present - yet! one of the guys is currently working on an update to ADT that will mean the "bundle" target results in a properly signed and notarized application or installer.

thanks

**Reply by QAthulhu** [reply]
Thanks, but I'm getting "Adobe AIR.dmg is damaged" error when trying to open the installer. Can anyone else confirm with Mac OS? Using Big Sur 11.1

**Reply by linxu1231** [reply]
> Thanks, but I'm getting "Adobe AIR.dmg is damaged" error when trying to open the installer. Can anyone else confirm with Mac OS? Using Big Sur 11.1

I got the same error using Catalina 10.15.

**Reply by ajwfrost** [reply]
Curious - I've just checked with Big Sur and the dmg file downloads fine, when I double-click it just opens up the folder, when I launch the application it gives me the standard warning about files downloaded from the internet..

![image](https://user-images.githubusercontent.com/6400804/105810081-3c8ac480-5fa2-11eb-8e8e-16e9af7a9062.png)

Can I check what's in your System Preferences -> Security & Privacy, it needs to allow apps from identified developers. Although if it's complaining about the dmg file itself, I'm wondering if that's the issue...

If you do `shasum -a 256 AdobeAIR.dmg` on the downloaded file, do you get the value from the bottom of the web page:
`eb13480e58d0fb5a7fd644d34c2a1aacb94792f5b7945d7baf6071e2bc713183`

**Reply by QAthulhu** [reply]
Tried downloading another copy today, and it worked fine, was able to install successfully. Perhaps something wrong with my connection yesterday. Everything seems to be working fine now. Thank you!
